### ISSUE TITLE
Extracting frames for prediction

### ISSUE THREAD

**Issue by JulianNeuberger** [assembly-101/assembly101-action-recognition#15]
Hi,

I am trying to use your pretrained model for predicting actions on the recordings you provide in Google Drive.
I downloaded the recordings and generated fine grained labels using [gen_fine_labels.py](https://github.com/assembly-101/assembly101-action-recognition/blob/main/gen_fine_labels.py), as you described.
When I run test_models.py, a `FileNotFoundError` is raised: It seems the dataloader is looking for frames, e.g., `nusar-2021_action_both_9011-a01_9011_user_id_2021-02-01_153724/C10095_rgb/C10095_rgb_0000000147.jpg`.

I tried creating these frames using the utilities from TSM (vid2img_kinetics.py, vid2img_sthv2.py), but they create frames with a mismatching file name (e.g., `img_16712.jpg`).
How can I fix this issue? Should I change the utilities of TSM, to produce the expected file names? Or am I missing a folder on Google Drive, that has the needed frames?

Thank you very much for your help with this issue!
